tools/cldr-apps/src/org/unicode/cldr/web/SurveyMain.java

```diff
@@ -370,9 +369,6 @@ public static String modifyThing(WebContext ctx) {
 
     public static String xMAIN = "general";
 
-    public static String CALENDARS_ITEMS[] = PathUtilities.getCalendarsItems();
```

Comment:
I search all Java files in the CLDR project and these strings aren't referenced anywhere.

Reply:
Actually they are referenced in menu_top.jsp. I'm not sure whether there is any ill-effect other than these "problems" reported by Eclipse:
```
CALENDARS_ITEMS cannot be resolved or is not a field	menu_top.jsp	/cldr-apps/WebContent/WEB-INF/tmpl	line 5	JSP Problem
METAZONES_ITEMS cannot be resolved or is not a field	menu_top.jsp	/cldr-apps/WebContent/WEB-INF/tmpl	line 6	JSP Problem
```

Ideally we should stop using jsp files entirely. In the meantime we should clean up menu_top.jsp by removing the useless references to CALENDARS_ITEMS and METAZONES_ITEMS.

Reply:
Ahh good spot, thanks. JSP strikes again.
I'll have a look and see about putting another PR together.